tools/rsync-to

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
```

Comment:
FWIW, this is script is a rough clone of a dozen or so of these in other repos. It has limitations but is useful for dev. In particular emoray builds don't currently bundle their test/... dir. This can rsync that test dir and devDependencies from the dev machine to COAL to run the test suite.